website/static/js/institutionProjectSettings.js

```diff
@@ -2,105 +2,189 @@
 var $ = require('jquery');
 var ko = require('knockout');
 var Raven = require('raven-js');
+var m = require('mithril');
 var bootbox = require('bootbox');
 
 var $osf = require('js/osfHelpers');
+var projectSettingsTreebeardBase = require('js/projectSettingsTreebeardBase');
 
 var ViewModel = function(data) {
     var self = this;
     self.loading = ko.observable(true);
     self.showAdd = ko.observable(false);
     self.institutionHref = ko.observable('');
     self.userInstitutions = window.contextVars.currentUser.institutions;
```

Comment:
Ah, looks like this won't be an issue. Henrique made the refactor in https://github.com/CenterForOpenScience/osf.io/pull/5744 to remove usage of contextVars, so you'll no longer have to set contextVars in your tests.

Can you please update your tests to reflect this?